dev-haskell/beam-automigrate/beam-automigrate-0.1.2.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="DB migration library for beam, targeting Postgres"
+HOMEPAGE="https://hackage.haskell.org/package/beam-automigrate"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="ghcipretty integration-tests werror examples"
+
+PATCHES=( "${FILESDIR}/${P}-add-examples-flag.patch" )
+
+RDEPEND=">=dev-haskell/aeson-1.4.4:=[profile?] <dev-haskell/aeson-1.5:=[profile?]
```

Comment:
This aeson dependency will cause an problem once #1118 is merged (or maybe I should say **if** it gets merged...)

Can you test with with the newer aeson, and maybe submit a PR upstream to bump their version?

Reply:
What is the best way to test this since the PR still isn't merged? If I upgrade aeson to 1.5, then I'll have a whole mess of broken packages, won't I?

Reply:
I can merge your `quickcheck_fixes` branch onto a copy of my `beam` branch, and see what happens. I may end up finding some fixes for your PR in the meantime.

Reply:
Probably merging into a copy is the best way, you're welcome to try this and more than welcome to point out any issues.

If you're not up to it, though, I don't mind testing it for you.

Reply:
A list of packages that need a dependency bump, just for safe-keeping:

depends on aeson:

- [ ] beam-core
- [ ] beam-migrate
- [ ] beam-postgres
- [ ] beam-sqlite
- [ ] beam-automigrate

depends on dlist:

- [ ] beam-core
- [ ] beam-sqlite
- [ ] beam-automigrate

depends on splitmix:

- [ ] beam-automigrate

Reply:
Ok, I should have this fixed: these packages no longer have an upper version constraint for these dependencies. This should make it future-proof in terms of #1118 getting merged. These packages have been tested with [that PR's branch](/ezzieyguywuf/gentoo-haskell/tree/quickcheck_fixes).